src/main/java/com/demo/program/exceptionhandling/Contact.java

```diff
@@ -0,0 +1,109 @@
+package exceptionhandling;
+
+import javax.activity.InvalidActivityException;
+
+public class Contact {
+	
+	String firstName, middleName, lastName, dateOfBirth, anniversary, address, area, city, state, country;
+	int pincode;
+	char gender;
+	String mobileNo, telePhoneNo, email, website;
+	
+	public Contact() {
+		
+	}
+	
+	public Contact(String email, String firstName, String lastName, String dateOfBirth, String mobileNo, String telePhone){
+		
+		this.firstName = firstName;
+		this.lastName = lastName;
+		this.email  = email;
+		this.mobileNo = mobileNo;
+		this.telePhoneNo = telePhone;
+		this.dateOfBirth = dateOfBirth;
+		
+	}
+		
+	void validate() throws PhoneNumberMissingException, InvalidEmailException, MandatoryAttributeException {
+		
+		String emailPattern = "^[\\w-\\+]+(\\.[\\w]+)*@[\\w-]+(\\.[\\w]+)*(\\.[a-z]{2,})$";
+		
+		if(this.firstName.equalsIgnoreCase("") || this.firstName == null){
```

Comment:
Add **null check** first and then **equal check** so it will prevent null pointer exception

Reply:
Ok, will do, thank you for reviewing!